flight/Modules/Airspeed/airspeed.c

```diff
@@ -104,10 +104,10 @@ static int8_t airspeedADCPin=-1;
 // Private functions
 static void airspeedTask(void *parameters);
 void baro_airspeedGet(BaroAirspeedData *baroAirspeedData, uint32_t *lastSysTime, uint8_t airspeedSensorType, int8_t airspeedADCPin);
-static void AirspeedSettingsUpdatedCb(UAVObjEvent * ev);
+static void AirspeedSettingsUpdatedCb(UAVObjEvent * ev, void *ctx, void *obj, int len);
```

Comment:
It's not the "default call".  It's the existing code.  People abuse the callback by putting the entirety of configuration update handler stuff in it (very resource intensive and potentially unsafe in multiple ways).  Then they want to invoke the handler directly because they're not guaranteed that the callback ever gets called or when that first call will happen.  So they pump a "fake event" with no details (NULL, NULL, NULL, 0).